src/api/hypixel.ts

```diff
@@ -16,10 +18,17 @@ const headers = {
 // Transformed responses will be stored in a database, not cached in memory like this
 
 export async function FetchPlayerData(uuid: string) {
+	if (IsUUID(uuid) === false) {
+		const playerUUID = await FetchMinecraftAccountByUsername(uuid);
+
+		uuid = playerUUID?.id || uuid;
+	}
```

Comment:
This uuid check and the other one should be changed to use the `FetchUuid` method instead